### PR TITLE
Datasource: Prefix SigV4 json data keys with sigV4

### PR DESCRIPTION
**Pull request by marefr**
**What this PR does / why we need it**:
Prefixes SigV4 json data and secure json data keys with `sigV4` to make sure they don't collide with any other settings. Also minor fixes to UI.

**Which issue(s) this PR fixes**:
Ref #27552 

**Special notes for your reviewer**:

### CODE REVIEW

File: packages/grafana-ui/src/components/DataSourceSettings/SigV4AuthSettings.tsx [hunk 1]
```diff
@@ -97,21 +97,17 @@ export const SigV4AuthSettings: React.FC<HttpSettingsProps> = props => {
             <Select
               className="width-30"
               value={authProviderOptions.find(
-                authProvider => authProvider.value === dataSourceConfig.jsonData.authType
+                authProvider => authProvider.value === dataSourceConfig.jsonData.sigV4AuthType
               )}
               options={authProviderOptions}
-              defaultValue={dataSourceConfig.jsonData.authType || 'keys'}
+              defaultValue={dataSourceConfig.jsonData.sigV4AuthType || 'keys'}
```
Comment (marefr) [inline packages/grafana-ui/src/components/DataSourceSettings/SigV4AuthSettings.tsx:103]: Fallback to default doesn't seem to work:
image

I also think that default should be "default"/AWS SDK Default

Comment (wbrowne) [reply packages/grafana-ui/src/components/DataSourceSettings/SigV4AuthSettings.tsx:103]: Weird.. must have got messed up from switching from the original design to the existing CW config editor. Looks like that's even changed again so we might be able to just make them consistent presuming CW works as expected.

Agree regarding the default 👍

**Review by wbrowne — APPROVED** [review]
Checked locally and it looks good 👍

File: packages/grafana-ui/src/components/DataSourceSettings/SigV4AuthSettings.tsx (hunk 1 above)
Comment (marefr) [reply packages/grafana-ui/src/components/DataSourceSettings/SigV4AuthSettings.tsx:103]: Okay I'll merge this one for now and handle changes in upcoming PR when CloudWatch auth changes have been merged

**Pull request merged**